browser/ephemeral_storage/ephemeral_storage_tab_helper.cc

```diff
@@ -12,6 +12,7 @@
 #include "base/hash/md5.h"
 #include "base/no_destructor.h"
 #include "base/optional.h"
+#include "base/threading/thread_task_runner_handle.h"
```

Comment:
It's not necessary to introduce a thread-bound task runner here, a sequence will do fine. It's generally better to use ```SequencedTaskRunner``` and signal other developers that your intentions are not related to an actual thread-bound operation (like some win32 COM-specific stuff).

https://github.com/chromium/chromium/blob/91.0.4450.1/docs/threading_and_tasks.md#posting-to-the-current-thread

Reply:
I was confused trying to understand what you were saying until I realized I had unintentionally used `ThreadTaskRunnerHandle` instead of `SequencedTaskRunner`  :)